addons/mass_mailing/models/mailing.py

```diff
@@ -670,6 +681,24 @@ def action_retry_failed(self):
         failed_mails.unlink()
         self.action_put_in_queue()
 
+    def action_view_link_trackers(self):
+        model_name = self.env['ir.model']._get('link.tracker').display_name
+        recipient = self.env['ir.model']._get(self.mailing_model_real).display_name
+        helper_header = _("No link tracker for that mailing!")
```

Comment:
Small detail: shouldn't we capitalize "Link Tracker" here as it's capitalized in the helper message?